GitUI/CommandsDialogs/FormCommit.cs

```diff
@@ -409,6 +404,8 @@ protected override void OnShown(EventArgs e)
                 Initialize();
             }
 
+            UpdateAuthorInfo();
+
```

Comment:
Needs to be activated event. OnShown runs once. If user alters name or email it will not stay current.

Reply:
I think he needs to close the form to update the settings and reopen it.
If he did it in an external console, I think that we could live with that...

Reply:
Thanks Jay.
I'm split both ways... I suppose we can call it from both places.

Reply:
Wrong. I developed that piece. I know what it does. Click the link to alter the user name or email. It WAS in activated event for a reason.

Reply:
Fixed

Reply:
👍   The issue is the user will click the link, change user name and save.  They close settings dialog.  Commit form activates and reads in new settings.  The code was in the activated event to catch user changing setting even from console and then creating issues because their info is not correct when they will commit.  When he changed the code from Activated to Application Activated, he changed that functionality.

Reply:
I have updated the code, please have a look.

Reply:
@RussKie Still OK for me.